Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.7
+
+WORKDIR /code
+COPY requirements.txt ./requirements.txt
```

Comment:
we kunnen dit vervangen door in een "pipeline" een venv aan te maken en dan gwn poetry install te doen.
Je hoeft dan niet manueel je dependancys bij te houden in je requirements file.
Anders moeten we hier eens over connecten?